webpack/assets/javascripts/react_app/routes/Audits/constants.js

```diff
@@ -11,4 +11,4 @@ export const AUDITS_PAGE_CLEAR_ERROR = 'AUDITS_PAGE_CLEAR_ERROR';
 export const AUDITS_PATH = '/audits';
 export const AUDITS_SEARCH_PROPS = getControllerSearchProps('audits');
 export const AUDITS_MANUAL_URL = version =>
-  `${getManualURL(version)}#4.1.4Auditing`;
+  `${getManualURL(version)}/4.1.4Auditing`;
```

Comment:
Perhaps the method should take an additional argument:
```suggestion
  getManualURL(version, '4.1.4Auditing');
```